tests/Geta.Net.Extensions.Tests/QueryStringBuilderTests.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Geta.Net.Extensions.Generators;
+using Xunit;
+
+namespace Geta.Net.Extensions.Tests
+{
+    public class QueryStringBuilderTests
+    {
+        [Fact]
+        public void AddStringParameterToAbsoluteUrl()
```

Comment:
Test naming convention is different in this project:
https://github.com/Geta/Net.Extensions/blob/7a8454d50d052ceb288e1a3cdbe3ba2e2c63c727/tests/Geta.Net.Extensions.Tests/ConfigurationHelperTests.cs
Please update test names according to it:
- separate words with an underscore
- start with the use case description
- end with result description